sfge/src/main/java/com/salesforce/graph/ops/GraphUtil.java

```diff
@@ -154,8 +156,17 @@ public static void loadSourceFolders(GraphTraversalSource g, List<String> source
             }
         }
 
+        final ProgressListener progressListener = ProgressListenerProvider.get();
```

Comment:
Could we have the listener be a `static final` property to avoid the need to keep declaring a variable for it?

Reply:
`static final` would force instantiation at the earliest and not use the lazy loading functionality as intended.

Reply:
https://www.imperva.com/learn/performance/lazy-loading/